featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1238,8 +1238,8 @@ def test_cluster_creation(self, monkeypatch):
                             get_mock_client_cluster)
         try:
             cpus = len(psutil.Process().cpu_affinity())
-        except AttributeError:
-            cpus = psutil.cpu_count()
+        except AttributeError:  # pragma: no cover
+            cpus = psutil.cpu_count()  # pragma: no cover
```

Comment:
Does this still need updating?

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/1829/commits/a1ac50e2b4fdb5c767f474154d97e198c66e0294